src/rateEngine/BillingDeterminantFactory.ts

```diff
@@ -60,6 +63,8 @@ class BillingDeterminantFactory {
         return new DemandTiersInMonths(args as BlockedTiersArgs, loadProfile);
       case 'DemandTimeOfUse':
         return new DemandTimeOfUse(args as DemandTimeOfUseArgs, loadProfile);
+      case 'DemandPerDay':
```

Comment:
good job remembering to add this to the factory! without naming names (cough... me...cough), we've definitely forgotten that a few times